org.cryptomator.Cryptomator.yaml

```diff
@@ -0,0 +1,100 @@
+app-id: org.cryptomator.Cryptomator
+command: cryptomator
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+separate-locales: false
+finish-args:
+  - --device=dri
+  - --env=PATH=/app/bin/:/app/jre/bin/:/usr/bin/
+  - --filesystem=home
+  - --share=ipc
+  - --share=network
+  - --socket=session-bus
+  - --socket=x11
+  - --talk-name=org.freedesktop.Flatpak
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.gtk.vfs.*
+  - --talk-name=org.gtk.vfs
+cleanup:
+  - /include
+  - /lib/pkgconfig
+modules:
+  - name: libfuse
+    config-opts:
+      - MOUNT_FUSE_PATH=/app/bin
+    post-install:
+      - install fusermount-wrapper.sh /app/bin/fusermount
+    sources:
+      - type: archive
+        url: https://github.com/libfuse/libfuse/releases/download/fuse-2.9.9/fuse-2.9.9.tar.gz
+        sha256: d0e69d5d608cc22ff4843791ad097f554dd32540ddc9bed7638cc6fea7c1b4b5
+      - type: file
+        path: build-aux/fusermount-wrapper.sh
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+  - name: cryptomator
+    buildsystem: simple
+    build-options:
+      env:
+        PATH: /app/bin:/usr/bin:/usr/lib/sdk/openjdk17/bin
+        MAVEN_OPTS: -Dmaven.repo.local=.m2/repository
+        JAVA_HOME: /usr/lib/sdk/openjdk17/jvm/openjdk-17
+        VERSION: 1.6.5
+        REVISION_NO: '1'
+    build-commands:
+      - mvn clean package -DskipTests -Plinux
+      - cp dist/linux/launcher* target
+      - cp target/cryptomator-*.jar target/mods
+      - cd target
+      - jlink
+        --output runtime
+        --module-path "${JAVA_HOME}/jmods"
+        --add-modules java.base,java.desktop,java.logging,java.naming,java.net.http,java.scripting,java.sql,java.xml,jdk.unsupported,jdk.crypto.ec,jdk.accessibility
+        --no-header-files
+        --no-man-pages
+        --strip-debug
+        --compress=1
+      - jpackage
+        --type app-image
+        --runtime-image runtime
+        --input target/libs
+        --module-path target/mods
+        --module org.cryptomator.desktop/org.cryptomator.launcher.Cryptomator
+        --dest .
+        --name Cryptomator
+        --vendor 'Skymatic GmbH'
+        --copyright '(C) 2016 - 2022 Skymatic GmbH'
+        --java-options '-Xss5m'
+        --java-options '-Xmx256m'
+        --java-options '-Dfile.encoding='utf-8''
+        --java-options '-Dcryptomator.logDir='~/.local/share/Cryptomator/logs''
```

Comment:
I can't remember if this will resolve to the actual user's home or will it try to resolve the `XDG_*` family.

Based on the history of the files that I've got, I would guess it will use the actual user's home, that's why I used something like this initially:

```shell
java --class-path "/app/Cryptomator/*" \
  -Dcryptomator.ipcPortPath="${XDG_CONFIG_HOME}/Cryptomator/ipcPort.bin" \
  -Dcryptomator.logDir="${XDG_DATA_HOME}/Cryptomator/logs" \
  -Dcryptomator.mountPointsDir="${XDG_DATA_HOME}/Cryptomator/mnt" \
  -Dcryptomator.settingsPath="${XDG_CONFIG_HOME}/Cryptomator/settings.json" \
  -Xmx512m \
  -Xss20m \
  org.cryptomator.launcher.Cryptomator
```

Reply:
Variable extension will not work in this case... those `--java-options` param end up in a properties file where environment variables have no effect. This may need an upstream fix to support those XDG dirs.